src/Core/System/Locale/LanguageLocaleCodeProvider.php

```diff
@@ -28,7 +28,15 @@ public function getLocaleForLanguageId(string $languageId): string
             throw new LanguageNotFoundException($languageId);
         }
 
-        return $languages[$languageId]['code'];
+        $language = $languages[$languageId];
+        $code = $language['code'];
+        $parentId = $language['parentId'];
+
+        if ($code === null && $parentId !== null) {
+            $code = $this->getLocaleForLanguageId($parentId);
+        }
+
+        return $code;
```

Comment:
please add this fallback for the `getLocalesForLanguageIds` too

Reply:
Yes, I'll fix this in the language loader. I was just not sure if some other dependencies of LanguageLoaderInterface::loadLanguages expect getting locale = null in case of a child language.